semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -121,7 +125,8 @@ public class HttpSemanticConventionUtils {
   private static final List<String> SCHEME_ATTRIBUTES =
       List.of(
           HttpSemanticConventions.HTTP_REQUEST_X_FORWARDED_PROTO.getValue(),
-          OTelHttpSemanticConventions.HTTP_SCHEME.getValue());
+          OTelHttpSemanticConventions.HTTP_SCHEME.getValue(),
+          HttpSemanticConventions.HTTP_REQUEST_ORIGIN.getValue());
```

Comment:
HTTP_REQUEST_ORIGIN seems to be getting handled specially in getHttpSchemeFromRawAttributes. Why do we need to add it to the list here?

Reply:
fixed